wlog/logger_provider_jsonmarshal.go

```diff
@@ -41,28 +41,34 @@ func (l *jsonMapLogger) Log(params ...Param) {
 func (l *jsonMapLogger) Debug(msg string, params ...Param) {
 	switch l.level {
 	case DebugLevel:
-		l.logOutput(append(params, StringParam("message", msg), StringParam("level", "DEBUG")))
+		params = append(params, StringParam("message", msg), StringParam("level", "DEBUG"))
+		ReverseParams(params)
+		l.logOutput(params)
 	}
 }
 
 func (l *jsonMapLogger) Info(msg string, params ...Param) {
 	switch l.level {
 	case DebugLevel, InfoLevel:
-		l.logOutput(append(params, StringParam("message", msg), StringParam("level", "INFO")))
+		l.logOutput(append([]Param{StringParam("message", msg), StringParam("level", "INFO")}, params...))
```

Comment:
Why does this logic for "Info" not match the others?

Reply:
Fixed, I think this is the correct logic